packages/voice/src/networking/VoiceUDPSocket.ts

```diff
@@ -128,16 +105,6 @@ export class VoiceUDPSocket extends EventEmitter {
 	 * @param buffer - The received buffer
 	 */
 	private onMessage(buffer: Buffer): void {
-		// Handle keep alive message
-		if (buffer.length === 8) {
-			const counter = buffer.readUInt32LE(0);
-			const index = this.keepAlives.findIndex(({ value }) => value === counter);
-			if (index === -1) return;
-			this.ping = Date.now() - this.keepAlives[index]!.timestamp;
```

Comment:
With this gone, wouldn't that make `ping` never assigned?

Reply:
🤷‍♂️ Yes, but there's nothing we can do about it, and removing it would be semver major

Reply:
Could at least deprecate it then?

Reply:
Just realised this package is in 0-ver and I don't believe any other subpackage depends on this package. It should be fine to wholly remove?